lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -47,6 +47,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
           return true;
         }
       })
+      .filter(anchor => {
+        // Ignore href's that do not redirect to a new url
+        return !/javascript:.*/.test(anchor.href);
```

Comment:
wouldn't this be better?
`return !/^javascript:/.test(anchor.href);` just to make sure the string starts with javascript:

Reply:
@wardpeet Thats a good idea. Makes it much more robust. Done!